BookEcommerce/Program.cs

```diff
@@ -21,18 +25,18 @@
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 builder.Services.AddAutoMapper(options => options.AddProfile(typeof(MapperProfile)));
-var localConnectionString = string.Empty;
-localConnectionString = builder.Configuration.GetConnectionString("LocalConnection");
 builder.Services.AddDbContext<ApplicationDbContext>(options =>
 {
-    options.UseSqlServer(localConnectionString, b => b.MigrationsAssembly("BookEcommerce"));
+    options.UseSqlServer("Server=DD-DATDH\\SQLEXPRESS;Database=BOOK_STORE_ECOMMERCE;Trusted_Connection=True;MultipleActiveResultSets=true;TrustServerCertificate=True", b => b.MigrationsAssembly("BookEcommerce"));
```

Comment:
use config to set connection string

Reply:
i've put it into config file but it still there, i will check it again

Reply:
Updated